Data/Gawade15

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Komal Gawade",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": " Learning Web developer ,  working on DSA , Exploring Technology",
+  "avatar":"https://github.com/Gawade15.png",
```

Comment:
Also `avatar` key is no longer needed, it will auto-pick from your GitHub profile.

```suggestion
```